.packit.yaml

```diff
@@ -4,8 +4,12 @@
 specfile_path: fido-device-onboard.spec
 
 files_to_sync:
-    - fido-device-onboard.spec
-    - .packit.yaml
+    - src:
+      - patches/0001-Revert-chore-use-git-fork-for-aws-nitro-enclaves-cos.patch
+      - .packit.yaml
+      - fido-device-onboard.spec
+      - "fido-device-onboard-rs-*-vendor-patched.tar.xz"
```

Comment:
Shouldn't this be `fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}-vendor-patched.tar.xz`?

Reply:
we don't have that variable exposed in this snippet - it's also documented https://packit.dev/docs/configuration/examples#custom-archive-creation-for-release-syncing (see also the thread linked in slack)